src/components/Icon/icons/Check.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export default function Check(props: any) {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink" viewBox="0 0 320 320" aria-hidden="true" {...props}>
+      <path id="a" d="M56.142 166.657c-7.81-7.816-20.474-7.816-28.284 0-7.81 7.815-7.81 20.487 0 28.303l68.13 68.178c7.812 7.816 20.475 7.816 28.285 0l151.87-151.973c7.81-7.815 7.81-20.487 0-28.303-7.81-7.816-20.475-7.816-28.285 0L110.13 220.682l-53.988-54.025z" />
```

Comment:
should delete the id in this path

Reply:
It's gone. I don't think I manually removed it -- I think SVGO did.